### PR TITLE
New Adapter: LM KiviAds

### PR DESCRIPTION
**Pull request by KiviAds**

### CODE REVIEW

File: static/bidder-info/lm_kiviads.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+endpoint: "http://pbs.kiviads.live/?pid={{.SourceId}}&host={{.Host}}"
```
Comment (onkarvhanumante) [inline static/bidder-info/lm_kiviads.yaml:1]: endpoint is reachable

```
% curl -i http://pbs.kiviads.live/
HTTP/1.1 200 OK
Server: nginx
Date: Fri, 04 Aug 2023 09:47:09 GMT
Transfer-Encoding: chunked
Connection: keep-alive

ok
```

File: static/bidder-info/lm_kiviads.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+maintainer:
+  email: "prebid@kiviads.com"
```
Comment (onkarvhanumante) [inline static/bidder-info/lm_kiviads.yaml:3]: Prebid team has sent verification mail on above email address.  Respond back on email with "received" message to complete verification

File: static/bidder-params/lm_kiviads.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "properties": {
+    "env": {
+      "type": "string",
+      "description": "kivi environment",
+      "minLength": 1
+    },
+    "pid": {
+      "type": "string",
+      "description": "Uniq placement ID",
+      "minLength": 1
```
Comment (onkarvhanumante) [inline static/bidder-params/lm_kiviads.json:15]: typo nit: `Uniq` should be `unique`

File: static/bidder-info/lm_kiviads.yaml (hunk 2 above)
Comment (KiviAds) [reply static/bidder-info/lm_kiviads.yaml:3]: done

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**